docs/xxgloballockreservation.md

```diff
@@ -0,0 +1,101 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]GlobalLockReservation
+
+**(Power only)**
+
+The `-XX:+GlobalLockReservation` option enables an optimization targeted towards more efficient handling of locking and unlocking Java&trade; objects.
+
+## Syntax
+
+        -XX:[+|-]GlobalLockReservation
+        -XX:+GlobalLockReservation:<parameter>
+
+| Setting                    | Effect | Default                                                                        |
+|----------------------------|--------|:------------------------------------------------------------------------------:|
+|`-XX:+GlobalLockReservation`| Enable |                                                                                |
+|`-XX:-GlobalLockReservation`| Disable| <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span> |
+
+This optimization is targeted towards applications with lots of uncontended locked objects that are being locked just to be safe. When enabled, heuristics are used to try and determine when an object will be exclusively locked by a single thread so that faster, more specialized code can be used for locking the object. Negative performance side effects may occur if the heuristics used get things wrong but functional correctness should still be maintained.
+
+The `-XX:-GlobalLockReservation` option turns off a previously enabled `-XX:+GlobalLockReservation` option.
+
+## Parameters
+
+| Parameter                                                     | Effect                                                                                          |
+|---------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| [`reservedTransitionThreshold`](#reservedtransitionthreshold) | Advanced tuning parameter. Changes amount of time spent analyzing an object.                    |
```

Comment:
Rather than stating *Advanced tuning parameter* for every row in this table, please can we include a lead in sentence before the table:

"Advanced tuning parameters are shown in the following table:"

Reply:
I can do this as well.